layouts/posts/single.html

```diff
@@ -2,5 +2,24 @@
   {{ .Title }} · {{ .Site.Title }}
 {{ end }}
 {{ define "content" }}
-  {{ partial "post.html" . }}
+  <section class="container post">
+    <article>
+      <header>
+        <h1 class="title">{{ .Title }}</h1>
+        <h2 class="date">{{ .Date.Format (.Site.Params.dateFormat | default "January 2, 2006" ) }}</h2>
+      </header>
+
+      <div>
+        {{ .Content }}
+      </div>
+
+      <br />
```

Comment:
This `<br />` is really necessary? All alignment should be done with CSS rules.

Reply:
Not really, it was like this before and I kept it. I'll adjust the css rules for it.

Reply:
No problem. You can leave it to another PR if you want.

Reply:
I updated this PR anyway with css rule instead of `<br />`.